rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -140,7 +140,7 @@ void addSignature_fedPubKey_belongs_to_active_federation() throws Exception {
             createHash3(1).getBytes()
         );
 
-        verify(provider, times(1)).getPegoutsWaitingForSignatures();
```

Comment:
because now we are checking if signatures are null and `[returning early in that case](https://github.com/rsksmart/rskj/blob/ae68f5ef35e0f0fe7a6ae77a75f91c4c35a60836/rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java#L1576)`.

Reply:
Thanks. Weird test anyway, not sure if that's what it is intended to test. At least it's not what the name says.

But we can review this at a later stage

Reply:
Thanks. Weird test anyway, not sure if that's what it is intended to test. At least it's not what the name says.

But we can review this at a later stage

Reply:
yep, all these tests are a bit weird. It will make sense to take a deeper look at them eventually

Reply:
i created a task to do it